docs/how-to-guides/data-basics/4-building-a-create-page-form.md

```diff
@@ -0,0 +1,373 @@
+			/>
+	);
+}
+```
+
+#### onSave, onCancel
+
+In the `EditPageForm`, we dispatched the `saveEditedEntityRecord('postType', 'page', pageId )` action to save the edits that lived in the Redux state.
+
+In the `CreatePageForm`, however, we do not have any edits in the Redux state, nor we do have any `pageId`. The action we need to dispatch in this case is called [`saveEntityRecord`](https://developer.wordpress.org/block-editor/reference-guides/data/data-core/#saveentityrecord) (without the word _Edited_ in the name) and it accepts an object representing the new entity record instead of a `pageId`.
+
+The data passed to `saveEntityRecord` is sent via a POST request to the appropriate REST API endpoint. For example, dispatching the following action:
+
+```js
+saveEntityRecord( 'postType', 'page', { title: "Test" } );
+```
+
+Triggers a POST request to the [`/wp/v2/pages` WordPress REST API](https://developer.wordpress.org/rest-api/reference/pages/) endpoint with a single field in the request body: `title=Test`.
+
+Now that we know more about `saveEntityRecord`, let's use it in `CreatePageForm`.
+
+```js
+export function CreatePageForm( { onSaveFinished, onCancel } ) {
+	// ...
+	const { saveEntityRecord } = useDispatch( coreDataStore );
+	const handleSave = async () => {
+		const savedRecord = await saveEntityRecord(
+			'postType',
+			'page',
+			{ title }
+		);
+		if ( savedRecord ) {
+			onSaveFinished();
+		}
+	};
+	return (
+		<PageForm
+			{ /* ... */ }
+			onSave={ handleSave }
+			onCancel={ onCancel }
+			/>
+	);
+}
+```
+
+There is one more detail to address: our newly created pages are not yet picked up by the `PagesList`. Accordingly to the REST API documentation,  the `/wp/v2/pages` endpoint creates pages with `status=draft` by default, but returns pages with `status=publish`. The solution is to pass the `status` parameter explicitly:
+
+```js
+export function CreatePageForm( { onSaveFinished, onCancel } ) {
+	// ...
+	const { saveEntityRecord } = useDispatch( coreDataStore );
+	const handleSave = async () => {
+		const savedRecord = await saveEntityRecord(
+			'postType',
+			'page',
+			{ title, status: 'publish' }
+		);
+		if ( savedRecord ) {
+			onSaveFinished();
+		}
+	};
+	return (
+		<PageForm
+			{ /* ... */ }
+			onSave={ handleSave }
+			onCancel={ onCancel }
+			/>
+	);
+}
+```
+
+Go ahead and apply that change to your local `CreatePageForm` component, and let’s tackle the remaining two props.
+
+#### lastError, isSaving
+
+The `EditPageForm`  retrieved the error and progress information via the `getLastEntitySaveError`  and `isSavingEntityRecord` selectors. In both cases, it passed the following three arguments: `( 'postType', 'page', pageId )`.
+
+In `CreatePageForm`, however, we do not have a `pageId`. What now? We skip the `pageId` argument to retrieve the information about the entity record without any id – which is the newly created one. The `useSelect` call is thus very similar to the one from `EditPageForm`:
```

Comment:
```suggestion
In `CreatePageForm` however, we do not have a `pageId`. What now? We can skip the `pageId` argument to retrieve the information about the entity record without any id – this will be the newly created one. The `useSelect` call is thus very similar to the one from `EditPageForm`:
```